src/dfi/rpc_accounts.cpp

```diff
@@ -783,9 +783,31 @@ UniValue utxostoaccount(const JSONRPCRequest &request) {
         }
     }
 
+    const UniValue &txInputs = request.params[2];
```

Comment:
txInputs is unused and seems to be so completely for utxostoaccount!

Reply:
good catch. yes originally I wanted to add that too, but honestly forgot about it. 
Yes its completely ignored in this call, IMHO a bug in the original code.